apps/network-alerts/js/attention/attention.js

```diff
@@ -31,26 +30,24 @@ function sendNotification() {
   }
 
   return new Promise(function(resolve, reject) {
-    var title = params.title;
-    var body = params.body;
+    var titleL10n = params.title;
+    var bodyL10n = { raw: params.body };
     var req = navigator.mozApps.getSelf();
 
     req.onsuccess = function onsuccess(event) {
       var app = event.target.result;
 
-      var notification = new Notification(
-        navigator.mozL10n.get(title), {
-          body: body,
-          tag: '' + Date.now(), // needs to be unique
-          icon: NotificationHelper.getIconURI(app) + '?titleID=' + title
-        }
-      );
-
-      notification.onerror = function onerror() {
-        reject(new Error('CMAS: notification API error'));
-      };
-
-      notification.onshow = resolve;
+      NotificationHelper.send(titleL10n, { 
+        bodyL10n: bodyL10n,
+        tag: '' + Date.now(), // needs to be unique
+        icon: NotificationHelper.getIconURI(app) + '?titleID=' + titleL10n
```

Comment:
Oh! That's a great point. @hkasemir - I just realized that `closeOnClick` means not only that we don't need to manually close notifications in `onclick` but we also have to pass `closeOnClick: false` everywhere where the old code *did not* close on click to preserve the original behavior.

Can you please review your patches including the ones we already landed to check if we need to add `closeOnClick: false` to them? If so, can you file a bug and write one patch for all apps to bring back the behavior? I'll be your reviewer.